File: commands/teamCommands/help.js
const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');

module.exports = {
    data: new SlashCommandBuilder()
        .setName('help')
        .setDescription('List all available commands'),
        async execute(interaction){
            await interaction.reply(`Here is a list of commands for you to use:
            \t\`/rc\` - Use if you need to talk with Race Control.
            \t\`/help\` - Use to bring up a list of commands.
            \t\`/sheet\` - Use to bring up the Race Control Decision Sheet link.
            \t\`/protest\` - Use to log a new protest.
            \t\`/served\` - Use to notify Race control that you served a penalty.
            \t\`/clear\` - Use to request a cleared black flag.
            \t\`/tow\` - Use to request a tow.`);
        }
}

File: Commands/Help.cs
using Discord.Interactions;

namespace RaceControlBot.Commands
{
    public class HelpCommand()
        : InteractionModuleBase<SocketInteractionContext>
    {

        [SlashCommand("help", "Shows a list of command")]
        public async Task HelpAsync()
        {
            await DeferAsync();
            string helpText =
                @"Here is a list of commands for you to use:
                `/rc`     - Use if you need to talk with Race Control.
                `/help`   - Use to bring up a list of commands.
                `/sheet`  - Use to bring up the Race Control Decision Sheet link.
                `/protest`- Use to log a new protest.
                `/served` - Use to notify Race Control that you served a penalty.
                `/clear`  - Use to request a cleared black flag.
                `/tow`    - Use to request a tow.";

            await FollowupAsync(helpText);
        }
    }
}
